reana-ui/src/pages/workflowList/WorkflowList.js

```diff
@@ -66,16 +66,20 @@ function Workflows() {
 
     if (!interval.current && reanaToken) {
       interval.current = setInterval(() => {
+        const showLoader = false;
         dispatch(
-          fetchWorkflows({ ...pagination }, searchFilter, statusFilter, sortDir)
+          fetchWorkflows(
+            { ...pagination },
+            searchFilter,
+            statusFilter,
+            sortDir,
+            showLoader
```

Comment:
I think we can pass directly `showLoader=false` here.

Reply:
Yeah, had some doubts about this.. Initially I passed it directly, but it kind of looks weird when you read the code. False value does not say much about what is happening, that's why added the variable. Don't know, we can remove it

Reply:
Yeah definitely, what I mean is that you can pass the name of the variable as well. Like this:

```js
fetchWorkflows(
  { ...pagination },
  searchFilter,
  statusFilter,
  sortDir,
  showLoader=false
)
```

Correct me if I'm wrong, but I think it's allowed.

Reply:
Hm, it's not allowed (yet ;D). Maybe you are confusing it with [default parameters](http://es6-features.org/#DefaultParameterValues)?. Or another option would be to wrap everything into object and pass to `fetchWorkflows`.. Maybe let's leave it for now

Reply:
🤔 Maybe it's a new feature...

![image](https://user-images.githubusercontent.com/8863238/95051466-93b41300-06ed-11eb-8b2b-1a56457c887a.png)

Nevermind, it's not important.